src/LtiAssignmentsGradesService.php

```diff
@@ -98,6 +98,17 @@ public function createLineitem(LtiLineitem $newLineItem): LtiLineitem
         return new LtiLineitem($createdLineItem['body']);
     }
 
+    public function deleteLineitem(): array
+    {
+        $request = new ServiceRequest(
+            ServiceRequest::METHOD_DELETE,
+            $this->getServiceData()['lineitem'],
+            ServiceRequest::TYPE_DELETE_LINEITEM
+        );
```

Comment:
The other requests here have `$request->setContentType()->setAccept()`. Any reason it's not included here?

Reply:
This defaults to `application/json` in `ServiceRequest.php`. The body was still `null` when I tried the existing `CONTENTYPE` constants and I didn't see anything in the docs LTI or Canvas docs indicating that anything would be returned in the body. So, I figured no-op code was more confusing than not matching the other requests.